server/src/main/java/dev/slimevr/vr/trackers/IMUTracker.java

```diff
@@ -456,13 +456,15 @@ protected void adjustToIdentity(Quaternion store) {
 	}
 
 	private void fixGyroscope(Quaternion sensorRotation) {
+		sensorRotation = sensorRotation.clone();
 		sensorRotation.fromAngles(0, sensorRotation.getYaw(), 0);
 		gyroFix.set(sensorRotation).inverseLocal();
```

Comment:
```suggestion
		gyroFix.set(sensorRotation.inverseLocal());
```
If you want to be consistent with other changes.